app/views/pages/home.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">It works! 🎉</h1>
+
+    <p class="govuk-body">
+      Your application is ready - so long as this page rendered without any errors you're good to go.
+    </p>
+
+    = configuration_summary_list
+  </div>
+</div>
```

Comment:
This needs converting to haml!

Reply:
you are correct!

Reply:
Actually, it needed deleting - I missed that the template added it's own 'pages' for a home page (deleted the controller but missed this view) - deleted now!